ext/build_defaults.yaml

```diff
@@ -24,6 +24,9 @@ build_msi:
   hiera:
     ref: 'refs/tags/1.3.4'
     repo: 'git://github.com/puppetlabs/hiera.git'
+  mcollective:
+    ref: 'refs/tags/2.7.0'
+    repo: 'git@github.com:puppetlabs/marionette-collective.git'
```

Comment:
It's a key based way of cloning, so it works, but you're right in that it should be `git://github.com/...`